e2e/vue3-vite-e2e/tests/library.spec.ts

```diff
@@ -8,10 +9,6 @@ jest.setTimeout(60000);
 
 describe('library e2e', () => {
   it('should create library', async () => {
-    // Create app
-    const app = uniq('vue3-vite');
-    await runNxCommandAsync(`generate nx-vue3-vite:app ${app}`);
```

Comment:
Remove the app-generation in the e2e test for library generator.
I think we don't really need to generate an app before generate a library.

Reply:
Sounds good! I think the only reason to test app+lib together is to make sure they interact correctly (build, serve, etc). But most of the E2E tests don't need that.